public/docs/_examples/hierarchical-dependency-injection/dart/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+# #docregion
+name: 'hierarchical_di'
+version: 0.0.1
+description: An absolute bare-bones web app.
+
+environment:
+  sdk: '>=1.0.0 <2.0.0'
+
+dependencies:
+  angular2: '2.0.0-beta.1'
+  dart_to_js_script_rewriter: '^0.1.0'
+
+transformers:
+- angular2:
+    platform_directives:
+    - 'package:angular2/common.dart#CORE_DIRECTIVES'
+    - 'package:angular2/common.dart#FORM_DIRECTIVES'
```

Comment:
I'd recommend including the minimum necessary. It does not look like you need the form directives for this example app.